rholang/src/main/scala/rholang/rosette/Roselang.scala

```diff
@@ -306,8 +306,9 @@ extends AllVisitor[VisitorTypes.R,VisitorTypes.A] {
       val procTerm: StrTermCtxt = p.proc_.accept(this, collectBindings(bindingsResults))._1
       val (chanTerms, ptrnTerms, quotedPtrnTerms, productFreshes, unificationFreshes, wildcards) = toListOfTuples(bindingsResults)
       val consumeTerm = B("consume")(TS, B(_list, chanTerms), B(_list, wildcards), B(_list, quotedPtrnTerms), Tag("#f")) // #f for persistent
+      val wrappedPtrnTerms = ptrnTerms map (x => B(_list)(x))
```

Comment:
Hmm this seems to break token.rho. Let me debug why.

Reply:
For starters, token.rho is still working with 37d61546c609a63d4c31be15677c54f3cdc9c15f .

Reply:
Nvm I just needed to also remove the lists.